src/initialization/taylormode.jl

```diff
@@ -22,13 +22,22 @@ function initial_update!(integ, cache, init::TaylorModeInit)
             @assert df isa ArrayPartition
             df = df[2, :]
         end
-        pmat = f.mass_matrix * Proj(o)
+        # pmat = f.mass_matrix * Proj(o)
+        @assert f.mass_matrix === I
```

Comment:
Make things work with other mass matrices again. At least the way it was before, but maybe this is also a good opportunity to make sure that we do the correct thing with mass matrices.

Reply:
I restored the previous behaviour and so it "works" again, but I think this is not actually the correct thing we want to have.